README.md

```diff
@@ -7,45 +7,71 @@ If you have ***any questions*** please
 
 ## Overview
 
-This diagram is meant to give complete beginners an overview of our stack:
-(_**click** the image **to enlarge**_)
+The following diagram is an overview of our stack:  
 
 ![dwyl-stack-lighter-blue](https://cloud.githubusercontent.com/assets/194400/9354261/e971fe58-4666-11e5-848f-67dc11d41bfb.jpg)
+(_**click** the image **to enlarge**_)
 
-Each of these components will be described in detail below:
+<small>To edit/improve this diagram: https://github.com/dwyl/technology-stack/issues/1 </small>
 
-## Open Source Modules We Use
+We have produced a ***complete beginners guide*** for *each* of the components in our stack. (see below)
 
-### For Us By Us
+## Open Source Projects We Use
 
-We ***craft code*** to ***scratch our own itch*** and ***everything*** we do is ***always Open Source***
+### For Us *By* Us
+
+We ***craft code*** to [***scratch our own itch***](https://github.com/dwyl/start-here#our-approach-scratching-your-own-itch) and ***everything*** we do is ***always Open Source***
 
 | Project | Use Case | Build | Coverage | Dependencies | Learn |
 | --------|----------|:-----:|:--------:|:------------:|-------|
 | **[hapi-auth-jwt2]** | Authorization | [![build][jwt2-bi]][jwt2] | [![cov][jwt2-cc]][jwt2] | [![deps][jwt2-dep]][jwt2] | [learn-jwt] |
-| **[env2]** | Environment Variables | [![build][env2-bi]][env2] | [![cov][env2-cc]][env2] | [![deps][env2-dep]][env2] | [learn-env2] |
+| **[env2]** | Environment Variables | [![build][env2-bi]][env2] | [![cov][env2-cc]][env2] | [![deps][env2-dep]][env2] | [learn-env] |
```

Comment:
Dependencies badges appear to be broken in the view:
<img width="885" alt="broken dependencies links" src="https://cloud.githubusercontent.com/assets/4185328/9938069/75699a1e-5d5b-11e5-9a70-31571ddb5318.png">